src/test/java/com/fasterxml/jackson/databind/ext/MiscJavaXMLTypesReadWriteTest.java

```diff
@@ -121,6 +128,16 @@ public void testQNameDeser() throws Exception
         assertEquals("", qn.getLocalPart());
     }
 
+    @Test
+    public void testQNameDeserWithValueOfFeatureDisabled() throws Exception
+    {
+        String qstr = a2q("{'namespaceURI':'http://abc','localPart':'tag','prefix':'prefix'}");
+        QName qn = MAPPER.disable(DeserializationFeature.READ_QNAMES_USING_VALUE_OF).readValue(qstr, QName.class);
```

Comment:
Construct a new mapper, changing of shared `MAPPER` should not be done (behavior not defined, concurrent unit test could fail etc)